contrib/coordinator-statsd.py

```diff
@@ -59,22 +66,25 @@ async def report_reservations(session, statsd_client, tags):
             groups = {"": {}}
 
         for group_name, group in groups.items():
-            path = (
-                ["reservations", group_name]
-                + [group.get(t, "") for t in tags]
-                + [state]
+            inc_gauge(
+                gauges,
+                ".".join(
+                    ["reservations", group_name]
+                    + [group.get(t, "") for t in tags]
+                    + [state]
+                ),
             )
-            statsd_client.set(".".join(path), token)
 
 
-async def report_places(session, statsd_client):
+async def report_places(session, tags, gauges):
```

Comment:
There are still calls to `report_places()` with 2 arguments left in the code.

Reply:
Ah, ya. The `async def report()` isn't used anymore (in favor of `asyncio.gather` and I forgot to remove it :)